examples/units_and_coordinates/STEREO_SECCHI_starfield.py

```diff
@@ -24,16 +25,15 @@
 
 import sunpy.map
 from sunpy.coordinates import get_body_heliographic_stonyhurst
-from sunpy.net import helioviewer
+from sunpy.time import parse_time
 
 ###############################################################################
 # Let's download a STEREO-A SECCHI COR2 image from Helioviewer which provide
 # pre-processed images and load it into a Map.
+# To get data from Helioviewer, we will use their offical Python binding, ``hvpy``.
 
-hv = helioviewer.HelioviewerClient()
-f = hv.download_jp2('2014/05/15 07:54', observatory='STEREO_A',
-                    instrument='SECCHI', detector='COR2')
-cor2 = sunpy.map.Map(f)
+cor2_file = hvpy.save_file(hvpy.getJP2Image(parse_time('2014/05/15 07:54').datetime, hvpy.DataSource.COR2_A.value), "COR2.JPEG2000")
```

Comment:
The `.value` is a bug in `hvpy` that is being worked on.

Reply:
There is no Fido client in the first release. Until that occurs, we have to do it this way.

Reply:
I was more thinking can we get the file from the VSO via. Fido?

Reply:
Oh, maybe, I will have a look.

Reply:
If I try to download and use the VSO image, the figure doesn't work, its just blank. I wonder if the helioviewer image is "prep" where as the VSO one isnt.

Reply:
Changed it for now.